Makefile

```diff
@@ -3,9 +3,14 @@ export KUBEBUILDER_ASSETS ?= ${HOME}/.kubebuilder/bin
 
 ## Inject the app version into project.Version
 LDFLAGS ?= -ldflags=-X=github.com/aws/karpenter/pkg/utils/project.Version=$(shell git describe --tags --always)
+CLOUD_PROVIDER ?= aws
 GOFLAGS ?= -tags=$(CLOUD_PROVIDER) $(LDFLAGS)
 WITH_GOFLAGS = GOFLAGS="$(GOFLAGS)"
 
+## Values for applying Karpenter locally
+AWS_DEFAULT_REGION ?= us-west-2
```

Comment:
You should maintain these values in your own personal environment, since many developers on the project don't use this region

Reply:
Is there a better region to pick as the default? `us-east-1`. Seems useful to me that this would be set for users already when they are configuring development.